docs/en_US/Development/GameAPI/Basic.md

```diff
@@ -29,6 +30,19 @@ You will come across them frequently in subsequent documentation.
 
 <br>
 
+## 🧩 Plugin Object
+
+| Parameter       | Meaning                              | Data Type  |
+| ------------ | ------------------------------------- | --------- |
+| pl.name    | Plugin name                             | `String` |
+| pl.desc    | Plugin description                      | `String` |
+| pl.ver     | Plugin version (array)                  | `Array<Integer,Integer,Integer>` |
```

Comment:
`pl` may be confused with Player
and `plugin.ver` dosn't match what is written in your code

Reply:
> `pl` may be confused with Player and `plugin.ver` dosn't match what is written in your code

**Fixed!**